inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -148,6 +250,27 @@ class XmlSerializer : public ngraph::AttributeVisitor {
         m_xml_node.append_attribute(name.c_str())
             .set_value(create_atribute_list(adapter).c_str());
     }
+    void on_adapter(
+        const std::string& name,
+        ngraph::ValueAccessor<std::shared_ptr<Function>>& adapter) override {
+        pugi::xml_document xml_body;
+
+        ngfunction_2_irv10(xml_body, m_bin_data, *adapter.get(), m_custom_opsets);
+
+        if (name == "body") {
+            xml_body.first_child().remove_attribute("name");
+            xml_body.first_child().remove_attribute("version");
+            xml_body.first_child().set_name(name.c_str());
+            // TI, Loop do not have attributtes as regular ops, it is necessary to append "body"
+            // to layer above (m_xml_node.parent()) as in ngfunction_2_irv10() layer (m_xml_node) with empty attributes
+            // is removed.
+            m_xml_node.parent().append_copy(xml_body.first_child());
+        } else if (name == "net") {
+            m_xml_node.append_copy(xml_body.first_child());
```

Comment:
Why do we need  to call this `append_copy()`?

Reply:
It copies xml_body (which contains nGraph function) into empty XmlSerializer::m_xml_node.

Reply:
ngfunction_2_irv10 API is changed, hence no more copy is required.